Packs/Exterro/Integrations/Exterro/Exterro.py

```diff
@@ -0,0 +1,76 @@
+# python 3.9 imports
+from json import JSONDecodeError
+from traceback import format_exc
+
+# accessdata imports
+from accessdata.client import Client
+
+# xsoar imports
+import demistomock as demisto
+from CommonServerPython import *  # noqa # pylint: disable=unused-wildcard-import
+from CommonServerUserPython import *
+
+
+def _trigger_workflow(client, **kwargs):
+    result = client.connect.trigger(**kwargs)
+    if result.get("Status") is not True:
+        raise ValueError("Failed to trigger automation workflow.", result.get("Status"))
+
+    return CommandResults(outputs_prefix='Accessdata.Workflow',
+                          outputs_key_field='Status',
+                          outputs=result)
+
+
+def _test_module(client):
+    # test the client can reach the case list
+    try:
+        client.cases
+    except JSONDecodeError as exc:
+        raise RuntimeError('False API key provided to FTK Connect.', exc)
+    except DemistoException as exc:
+        raise RuntimeError('Authentication with FTK Connect failed.', exc)
+
+    return "ok"
+
+
+def main():
```

Comment:
```suggestion
def main():  # pragma: no cover
```
I see the build is failing due to low coverage percentage (you need at least 70 %).
Please add this comment to the main function and add tests for the **_test_module** function.

That's all from my side.